src/main/java/objects/character/Player.java

```diff
@@ -28,10 +32,11 @@ public class Player extends Character {
     private SkillType skillType;
     private int speed;
     private int money;
+    private final ArrayList<Skill> skillList;
```

Comment:
Player Instance attributes look good, although we will most likely be adding more stats in the future